build.gradle

```diff
@@ -201,7 +201,8 @@ subprojects {
         tasks.named("jar").configure {
             manifest {
                 attributes('Implementation-Title': name,
-                        'Implementation-Version': project.version)
+                        'Implementation-Version': project.version,
+                        'Automatic-Module-Name': 'io.grpc.' + project.name.substring(5)) // default name inferred from artifact
```

Comment:
I'm pretty wary of this. This is part of our API; I don't think it should be inferred. It's especially likely to be wrong for names that contain a dash (as we see here), and it won't be clear it has been forgotten when creating a new project.

Reply:
So you prefer each one to be explicitly specified in the respective gradle file. Okay